index.js

```diff
@@ -224,6 +224,25 @@ class ScmBase {
         return Promise.reject(new Error('Not implemented'));
     }
 
+     /**
+     * Decorate the author for the specific source control
+     * @method decorateAuthor
+     * @param  {Object}    config
+     * @param  {String}    config.username        Author to decorate
+     * @param  {String}    config.token           The token used to authenticate to the SCM
+     * @param  {String}    [config.scmContext]    The scm context name
+     * @return {Promise}
+     */
+    decoratePR(config) {
+        return validate(config, dataSchema.plugins.scm.decoratePR)
+            .then(validPR => this._decoratePR(validPR))
+            .then(decoratePR => validate(decoratePR, dataSchema.core.scm.pr));
+    }
+
+    _decoratePR() {
+        return Promise.reject(new Error('Not implemented'));
```

Comment:
If we reject it here, we need to make sure `scm-bitbucket` also implements this method with a dummy return value.

Reply:
yeah, I'm implementing it in all the other scm repos